example_template_tag_test.go

```diff
@@ -0,0 +1,34 @@
+package faker_test
+
+import (
+	"fmt"
+
+	"github.com/go-faker/faker/v4"
+)
+
+type Address struct {
+	Street string `faker:"street_name"`
+	City   string `faker:"city"`
+}
+
+type UserProfile struct {
+	FirstName string `faker:"first_name"`
+	LastName  string `faker:"last_name"`
+	Domain    string `faker:"domain_name"`
```

Comment:
We should follow the field name or tag name?
```suggestion
	DomainName    string `faker:"domain_name"`
```
or
```suggestion
	Domain    string `faker:"domain"`
```